testsuite/MDAnalysisTests/utils/test_distances.py

```diff
@@ -325,7 +355,7 @@ def test_selfdist(self):
 
     def test_distarray(self):
         from MDAnalysis.lib.distances import distance_array
-        from MDAnalysis.lib.distances import transform_StoR, transform_RtoS
+        from MDAnalysis.lib.distances import transform_StoR
```

Comment:
your IDE just sorted the imports. It can't actually remove them. When you remove them just make sure the tests still pass.